GOVERNANCE.md

```diff
@@ -27,6 +27,9 @@ Community members are asked to abide by the following principles:
    Remember, it's very hard to convey meaning on a purely written medium,
    especially between people from different cultures, technical backgrounds,
    English proficiency levels, etc.
+   Where possible, we use inclusive language, but sometimes we have kept the
+   original terminology to preserve the meaning of the text. We are merely
+   *documenting* here, not taking any political stance.
```

Comment:
I agree with adding a note about this here, but the current phrasing is unnecessarily defensive. Also, these rules are about communicating with one another, as members of our community, not about the content of the pages. IMO we can edit this to something like the following:

```
Use inclusive language whenever possible and reasonable — for example,
[avoid assuming gender](https://en.wikipedia.org/wiki/Singular_they),
and refrain from racially or ethnically charged terms.
```